plasmapy/formulary/collisions.py

```diff
@@ -2106,14 +2146,18 @@ def __init__(
             :math:`μ` is the reduced mass. Cannot be negative.
 
         T_a : `~astropy.units.Quantity`, optional
-            The temperature of the test species. Only necessary if :math:`v_a` is not provided.
+            The temperature of the test particles. Only necessary if :math:`v_a` is not provided.
 
-        n_b : `~astropy.units.Quantity`
-            The number density of the background field particles in units convertible to :math:`\frac{1}{m^{3}}`
+        n_a : `~astropy.units.Quantity`
```

Comment:
Oh, good catch, it does look like that's the test particle density doesn't it...

I have a feeling that, as we add more cases, explaining which of these parameters are required will get cumbersome in the docs. So, I suggest dropping it here, adding the required parameters to the docstring of each attribute, and then using errors to ensure that the right parameters are set whenever an attribute is called.